routes/discord_genlink.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+const router = express.Router();
+router.use(express.json());
+const discordService = require('../services/discord_bot_service');
```

Comment:
please let's keep the order as follows:

library/objects imports first (i.e. `require` code)
global variables second (i.e. `const router = express.Router(); router.use(express.json());`
then main code